app/assets/stylesheets/greetings.scss

```diff
@@ -16,18 +17,13 @@
   }
 }
 
-#buttons {
-  align-items: center;
-
-  .start-button {
-    outline: none;
-    min-width: 120px;
-    background: linear-gradient(#fffdfd, #bbb6b6);
-    border-radius: 5px;
-    border-width: 1px;
-    border-color: #5f5a5a;
-    font-size: 15px;
-    line-height: 20px;
-    margin-right: 2%;
-  }
+.start-button {
+  outline: none;
+  min-width: 130px;
+  padding: 2px;
+  background: linear-gradient(#fffdfd, #bbb6b6);
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: #5f5a5a;
```

Comment:
You can use [border](https://developer.mozilla.org/es/docs/Web/CSS/border) property as `border: 1px solid #5f5a5a;` without having to specify border width, color and style in different properties 😋